addons/product/views/product_document_views.xml

```diff
@@ -92,6 +93,10 @@
                                         <div class="mt-2" invisible="res_model != 'product.product'">
                                             <field name="res_name" style="font-style: italic; text-decoration-line: underline;"/>
                                         </div>
+                                        <div invisible="active_lang_count &lt;= 1" class="mt-2" name="language">
+                                            <span class="me-4">Language</span>
+                                            <field name="lang" class="ms-2 border-bottom border-dark" style="width: 59%;" widget="selection"/>
```

Comment:
Why width 59%? :thinking: other border classes

Also it is > 100

Reply:
To make it consistent with the `Sales Visibility` field just above it. It looks like a workaround but I will find another approach for it.